### ISSUE TITLE
BUG: can't delete tab after deleting the page

### ISSUE THREAD

**Issue by benwoodward** [dive2Pro/roam-tabs#2]
Great plugin, thank you! 

Here's a bug I've found:

- Command-click on keyword to create a tab
- In the tab for that page, delete the page
- Go back to Daily Notes page
- Try to delete the tab for the now deleted page
- You can't delete it

**Comment by dive2Pro** [comment]
thank you for reporting this. It should be fixed in next version.

**Comment by danyalaytekin** [comment]
Hi @dive2Pro.  Thanks for this extension.  Would it be possible for the tab to close automatically after deletion?  I thought about making a new GitHub Issue before finding this related issue.

**Comment by dive2Pro** [comment]
> Hi @dive2Pro. Thanks for this extension. Would it be possible for the tab to close automatically after deletion? I thought about making a new GitHub Issue before finding this related issue.

  
Yes, it is possible, but the solution would be a little trickier since there are no APIs to do so.
